kong.config.sh

```diff
@@ -63,6 +63,20 @@ authConfig "template"
 PAYLOAD
 authConfig "device"
 
+
+(curl -o /dev/null $kong/apis -s -S -X POST \
+    --header "Content-Type: application/json" \
+    -d @- ) <<PAYLOAD
+{
+    "name": "image",
+    "uris": "/image",
+    "strip_uri": false,
+    "upstream_url": "http://imgm:5000"
```

Comment:
Here I think it'd be better if we use the service name itself, i.e. `http://image-manager:5000`